com.github.lruzicka.Needly.yaml

```
@@ -0,0 +1,113 @@
+# The YAML manifest for the Needle Flatpak
+#
+app-id: com.github.lruzicka.Needly
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: needly
+finish-args: 
+   - --share=ipc # For X11
+   - --socket=wayland # Enable communication with Wayland
+   - --socket=x11 # Alternatively, with X11
```

```suggestion
   - --socket=fallback-x11 # Alternatively, with X11
```

unless there is a good reason you need simultaneous X11 and wayland.

---

Ok. I did not know how to use these exactly and thought that both of the sockets (x11 and wayland) will make sure it works on both systems. I will change it to fit your suggestion.

---

The application uses `tkinter` and `--socket=wayland` with `--socket=fallback-x11` fail with `no display` error. When I only use `--socket=x11` it works even on wayland, so I am just using x11 socket for now.